pkg/operators/clustermanager/controllers/clustermanagercontroller/clustermanager_controller.go

```diff
@@ -131,11 +146,14 @@ func (n *clusterManagerController) sync(ctx context.Context, controllerContext f
 	clusterManager = clusterManager.DeepCopy()
 
 	config := hubConfig{
-		ClusterManagerName:                       clusterManager.Name,
-		ClusterManagerNamespace:                  helpers.ClusterManagerNamespace,
-		RegistrationImage:                        clusterManager.Spec.RegistrationImagePullSpec,
-		ClusterManagerWebhookSecret:              clusterManagerWebhookSecret,
-		ClusterManagerWebhookRegistrationService: fmt.Sprintf("%s-registration-webhook", clusterManager.Name),
+		ClusterManagerName:         clusterManager.Name,
+		ClusterManagerNamespace:    helpers.ClusterManagerNamespace,
+		RegistrationImage:          clusterManager.Spec.RegistrationImagePullSpec,
+		WebhookRegistrationSecret:  registrationWebhookSecret,
+		WebhookRegistrationService: fmt.Sprintf("%s-registration-webhook", clusterManager.Name),
+		WorkImage:                  "quay.io/open-cluster-management/work",
+		WebhookWorkSecret:          workWebhookSecret,
+		WebhookWorkService:         fmt.Sprintf("%s-work-webhook", clusterManager.Name),
```

Comment:
does this name actually need to be variable? Since it's inside of our namespace, the name can be constant, right?

Reply:
yes, i've changed this to constant